notebooks/official/prediction/pytorch_image_classification_with_prebuilt_serving_containers.ipynb

```diff
@@ -0,0 +1,883 @@
+{
```

Comment:
<p>&gt;&gt;from numeric classes (0,1,2...) to friendly strings</p><p> </p><p>This does not seem to be accurate. I downloaded the file, and it looks like a string ID "n01773797" to friendly string "garden_spider". </p><p> </p><p>Moreover, it's not clear to me where these ID and friendly values are coming from (is it from the training data? are all the values unique?) and what the benefit is to providing this file (it sounds optional). For example, if I didn't provide the file, what would be different in the prediction input/output? Would I get IDs instead of friendly names for the prediction output?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1592/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>The index to name mappings for the pretrained models are typically published by the same provider who publishes the models so the index_to_name.json is from PyTorch. </p><p>The idea of the mapping file is to map a string class to an ID. The ID is typically in numeric form and a form is expected by the handler. The description of the index_to_name.json is from PyTorch documentation. </p>